sensor_msgs/PointCloud2 cloud
std_msgs/Int64[] indices
